main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"log"
+
+	"github.com/raymundo/academy-go-q42021/router"
+)
+
+func main() {
+
+	srv := router.NewRouter()
+
+	err := srv.ListenAndServe()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+}
```

Comment:
Please avoid having unnecessary empty lines.
```suggestion
func main() {
	srv := router.NewRouter()

	err := srv.ListenAndServe()
	if err != nil {
		log.Fatal(err)
	}
}
```